cs-database/pom.xml

```diff
@@ -0,0 +1,372 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xmlns="http://maven.apache.org/POM/4.0.0"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <!--
+           (c) Copyright 2016 Hewlett-Packard Development Company, L.P.
+           All rights reserved. This program and the accompanying materials
+           are made available under the terms of the Apache License v2.0 which accompany this distribution.
+
+           The Apache License is available at
+           http://www.apache.org/licenses/LICENSE-2.0
+       -->
+    <groupId>io.cloudslang.content</groupId>
+    <artifactId>cs-database</artifactId>
+    <version>0.0.1-SNAPSHOT</version>
+    <packaging>jar</packaging>
+    <name>${project.groupId}:${project.artifactId}</name>
+    <description>Database integration</description>
+    <url>https://github.com/CloudSlang/cs-actions</url>
+    <licenses>
+        <license>
+            <name>The Apache License, Version 2.0</name>
+            <url>http://www.apache.org/licenses/LICENSE-2.0.txt</url>
+        </license>
+    </licenses>
+    <scm>
+        <connection>scm:git:https://CloudSlang/cs-actions.git</connection>
+        <developerConnection>scm:git:git@github.com:CloudSlang/cs-actions.git</developerConnection>
+        <url>https://github.com/CloudSlang/cs-actions.git</url>
+        <tag>master</tag>
+    </scm>
+    <developers>
+        <developer>
+            <name>Victor Ursan</name>
+            <email>victor.ursan@hpe.com</email>
+            <organization>Hewlett Packard Enterprise</organization>
+            <organizationUrl>http://www8.hp.com/us/en/software/enterprise-software.html</organizationUrl>
+        </developer>
+    </developers>
+
+    <repositories>
+        <repository>
+            <id>atlassian-3rdparty</id>
+            <name>Atlassian 3rdparty</name>
+            <url>https://maven.atlassian.com/3rdparty/</url>
+        </repository>
+        <repository>
+            <id>clojars</id>
+            <name>clojars 3rdparty</name>
+            <url>http://clojars.org/repo/</url>
+        </repository>
+    </repositories>
+
+    <distributionManagement>
+        <repository>
+            <id>ossrh</id>
+            <name>Maven Central staging repository</name>
+            <url>https://oss.sonatype.org/service/local/staging/deploy/maven2/</url>
+        </repository>
+        <snapshotRepository>
+            <id>ossrh</id>
+            <name>Maven Central snapshots repository</name>
+            <url>https://oss.sonatype.org/content/repositories/snapshots/</url>
+        </snapshotRepository>
+    </distributionManagement>
+
+    <dependencies>
```

Comment:
@victorursan Is this going to have it`s dependencies added to the parent pom as well? Coverage will update with the new tests and it might increase the overall coverage.